utils/sccli.py

```diff
@@ -56,6 +54,19 @@ def system(cmd):
     returncode = ret.returncode
     return out, err, returncode
 
+def get_registry_image_tag_defaults():
+    try:
+        with open('/etc/sysconfig/openshift_option') as fh:
```

Comment:
We need to put `/etc/sysconfig/openshift_option` in to a variable and then use it in the script. We are already using it 4 places. If we don't have time for that then we need to create an issue and track it there.